setup.cfg

```diff
@@ -31,6 +31,8 @@ install_requires =
     napari-plugin-engine>=0.1.4
     numpy
     aydin
+    napari-tools-menu
+    napari-assistant
```

Comment:
This line is optional. We do not use any function in the napari-assistant, but if it's a dependency, it gets installed together with the plugin and thus, makes it available from the GUI